recipes/wolfssl/all/conanfile.py

```diff
@@ -76,6 +82,10 @@ def _configure_autotools(self):
             "--enable-harden",
             "--enable-debug" if self.settings.build_type == "Debug" else "--disable-debug",
         ]
+        if self.options.opensslextra:
+            conf_args.extend(["--enable-opensslextra"])
+            if self.options.opensslall:
+                conf_args.extend(["--enable-opensslall"])
```

Comment:
@kirandivekar 
Would this work as well?
```suggestion
            "--enable-opensslall" if self.options.opensslallelse "--disable-opensslall",
            "--enable-opensslextra" if self.options.opensslextra else "--disable-opensslextra",
        ]
```